generators/app/templates/src/components/Spinner/components/loading.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+// eslint-disable-next-line import/no-named-as-default
```

Comment:
why is this needed?

Reply:
I'm having an issue where 'react-spinkit' has a named export called Spinner and eslint thinks that's an error.